### PR TITLE
Improve docs about log inspection

### PR DESCRIPTION
**Pull request by GeorgianaElena**
### Update 2

Fixes https://github.com/2i2c-org/infrastructure/issues/1692
Fixes https://github.com/2i2c-org/infrastructure/issues/1890

Bigger context https://github.com/2i2c-org/infrastructure/issues/1167

### Update 1

I've repurposed this PR to address https://github.com/2i2c-org/infrastructure/issues/1692. This means, I've created a new section for how to check logs with the deployer and/or kubectl at https://2i2c-pilot-hubs--3366.org.readthedocs.build/howto/troubleshoot/logs/kubectl-logs/.

The 30m checklist just links to that now, and it'd probably just make sense to live completely at https://compass.2i2c.org/projects/managed-hubs/support/#non-incident-response-process. I will move it there tomorrow.

<details>
<summary> Original PR description </summary>
I believe that if we ever want to move first-line support away from engineering, we should have such a checklist.
This one is far being complete, but I wanted to at least start it and give it a place to live.

Should be linked from https://compass.2i2c.org/projects/managed-hubs/support/#non-incident-response-process I believe, or maybe even live in the team-compass repo?

Felt it is technical enough to be in the infrastructure repo. What do others think?

Rendered at https://2i2c-pilot-hubs--3366.org.readthedocs.build/sre-guide/support/timeboxed-initial-ticket-evaluation/
</details>

### CODE REVIEW

File: docs/sre-guide/support/timeboxed-initial-ticket-evaluation.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
+# Initial timeboxed (30m) ticket resolution checklist
+
+In the [non-incident support response process](https://compass.2i2c.org/projects/managed-hubs/support/#non-incident-response-process), an initial 30m timeboxed ticket resolution process is documented.
+
+The support triagers use these 30m time interval to try an resolve a ticket, before opening a follow-up issue about it.
+
+The next sections represents an not-complete initial checklist that the support triager can follow in order to resolve the ticket or decide on opening a tracking issue about it, with the context they gained during this investigation.
```
Comment (sgibson91) [inline docs/sre-guide/support/timeboxed-initial-ticket-evaluation.md:7]:
```suggestion
The next sections represents an incomplete initial checklist that the support triager can follow in order to resolve the ticket or decide on opening a tracking issue about it, with the context they gained during this investigation.
```

**Review by sgibson91 — COMMENTED** [review]
Thank you for starting this Georgiana!

**Comment by damianavila** [comment]
This is great, @GeorgianaElena!
I do think we need to cross-link in the team compass, but we can do that after merging this one.

Btw, I added @2i2c-org/engineering so we can feedback from all the support triagers (cc @consideRatio and @yuvipanda who maybe did not see this PR yet).

cc @AIDEA775 who is going to shadow support pretty soon.

**Comment by GeorgianaElena** [comment]
### Update

I've repurposed this PR to address https://github.com/2i2c-org/infrastructure/issues/1692. This means, I've created a new section for how to check logs with the deployer and/or kubectl at https://2i2c-pilot-hubs--3366.org.readthedocs.build/howto/troubleshoot/logs/kubectl-logs/.

The 30m checklist just links to that now, and it'd probably just make sense to live completely at https://compass.2i2c.org/projects/managed-hubs/support/#non-incident-response-process. I will move it there tomorrow.

**Comment by GeorgianaElena** [comment]
Update 2:

This PR is purely about inspecting logs and the 30 support checklist now references it from the team-compass repo https://github.com/2i2c-org/team-compass/pull/781

File: docs/howto/troubleshoot/logs/kubectl-logs.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+## Look at logs via deployer sub-commands
+
+There are some `deployer debug` sub-commands that wrap up the most relevant `kubectl logs` arguments that allow conveniently checking logs with only one command.
+
+````{tip}
+You can export the cluster's and hub's names as environmental variables to directly use the copy-pasted commands in the sections below.
+
+```bash
+export CLUSTER_NAME=2i2c; export HUB_NAME=staging
```
Comment (sgibson91) [inline docs/howto/troubleshoot/logs/kubectl-logs.md:14]:
````suggestion
export CLUSTER_NAME=2i2c; export HUB_NAME=staging
```
````

I don't think the bash code section was closed properly

File: docs/howto/troubleshoot/logs/kubectl-logs.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+````
+
+### Look at hub component logs
+
+The JupyterHub components's logs can be fetched with the `deployer debug component-logs` command, ran for each hub component.
```
Comment (sgibson91) [inline docs/howto/troubleshoot/logs/kubectl-logs.md:19]:
```suggestion
The JupyterHub component's logs can be fetched with the `deployer debug component-logs` command, ran for each hub component.
```

File: docs/howto/troubleshoot/logs/kubectl-logs.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+deployer debug user-logs  $CLUSTER_NAME $HUB_NAME <username>
+```
+
+Note that you don't need the *escaped* username, with this command.
+
+## Look at logs via kubectl
+
+### Pre-requisites
+
+Get the name of the cluster you want to debug and export its name as and env vars. Then use the `deployer` to gain `kubectl` access into this specific cluster.
```
Comment (sgibson91) [inline docs/howto/troubleshoot/logs/kubectl-logs.md:79]:
```suggestion
Get the name of the cluster you want to debug and export its name as env vars. Then use the `deployer` to gain `kubectl` access into this specific cluster.
```

**Review by sgibson91 — APPROVED** [review]
🎉

**Pull request merged**